frontend/projects/valtimo/case/src/lib/components/case-assign-user/case-assign-user.component.scss

```
@@ -14,23 +14,78 @@
  * limitations under the License.
  */
 
-.container-fluid {
-  color: #959595;
+:host ::ng-deep .cds--popover-content {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)